client/components/main/index.jsx

```diff
@@ -3,10 +3,10 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
```

Comment:
I agree, fixed.

We can do cleanup in a dedicated PR in order to keep this one smaller, and easier to review.